File: Ejercicio_1/ejercicio1.js
// Tlanetzi Chavez
// 1 agosto 2021

// Crear una funcion que devuelva la tabla de multiplicar de un numero dado




function tablaMultiplicar(numero){
    //let resultado = "# Tabla del " + numero + " #";
    let resultado = `# Tabla del ${numero} # \n`;

    for(let multiplicador = 1; multiplicador <= 10; multiplicador++){
        //let multiplicacion = (i*numero);

        resultado += `${multiplicador} x ${numero} = ${numero*multiplicador} \n`
        
    }
    return resultado;
}

console.log(tablaMultiplicar(6))

File: Ejercicio_1/ejercicio1.cpp
// Tlanetzi Chavez
// 1 agosto 2021

// Crear una funcion que devuelva la tabla de multiplicar de un numero dado

#include <iostream>
#include <string>

using namespace std;

string tablas(int numero){
    string respuesta;

    respuesta = "# tabla del " + to_string(numero) + " # \n";
    for(int multiplicador = 1; multiplicador < 11; multiplicador++){
        respuesta += to_string(multiplicador) + " x " + to_string(numero) + " = " + to_string(multiplicador * numero) + "\n";
    }
    return respuesta;
}

int main(){

    int numero;

    cout << "Numero a multiplicar:";
    cin >> numero;

    cout << tablas(numero);


    return 0;
}